modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -11,15 +11,69 @@ Use the following procedure to add the {product-title} repositories to Image Bui
 .Prerequisites
 * Your build host meets the Image Builder system requirements.
 * You have installed and set up Image Builder and the `composer-cli` tool.
-* You have root user access to your build host.
+* You have root-user access to your build host.
```

Comment:
Would it be better to use the term superuser here ?

Reply:
Hmm, good question. I don't see anything in the IBM Style Guide about it, nor do we use it often in the OCP Docs. I'll see if someone can point me to guidance on docs use of the term. TBD

Reply:
@dhensel-rh We _can_ use it, but haven't. I do not think it is worth a sweep through the docs to update this term consistently, so I'll leave it. The note we have in our supplementary style guide is that the term is more common in Solaris documentation than Linux.